styleguide/source/_patterns/02-organisms/03-sections/02-card-row-ec/03-card-row-ec-five-up-left.twig

```diff
@@ -0,0 +1,9 @@
+<div class="layout layout_five-up-left">
+  <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
+  <div class="layout_item layout_item-container">
+    <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
+    <div class="layout_item">{% include 'molecules-card-image-landscape' %}</div>
+    <div class="layout_item layout_item-visible-desktop">{% include 'molecules-card-image-landscape' %}</div>
```

Comment:
Addressed this [here](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/138/commits/ba7c5a15df2c2dcab046b973cc785aa5912d0ad7)